src/main/java/hudson/plugins/sloccount/SloccountAreaRenderer.java

```diff
@@ -47,4 +45,27 @@ public final String generateURL(final CategoryDataset dataset, final int row, fi
     private NumberOnlyBuildLabel getLabel(final CategoryDataset dataset, final int column) {
         return (NumberOnlyBuildLabel)dataset.getColumnKey(column);
     }
+
+    /**
+     * Checks this instance for equality with an arbitrary object.
+     *
+     * @param obj  the object (<code>null</code> not permitted).
```

Comment:
`equals()` in Java does not explicitly define the behavior for `null`: https://docs.oracle.com/javase/7/docs/api/java/lang/Object.html#equals(java.lang.Object), though the result of comparing existing object and null us quite predictable... 

I would recommend adding a protective check in the beginning or `@Nonnull` annotations. The first option is IMHO better